MANIFEST.in

```diff
@@ -1,3 +1,2 @@
 include *.rst
-include *.txt
-include environ/test_env.txt
```

Comment:
`environ/test_env.txt` actually should be included. Files listed in MANIFEST.in are included in sdist tarbar file.